src/app/(frontend)/page.tsx

```diff
@@ -1,9 +1,10 @@
 import { fileURLToPath } from 'url'
-import React from 'react'
+import React, { Suspense } from 'react'
 import { headers as getHeaders } from 'next/headers.js'
 import Image from 'next/image'
 import { getPayload } from 'payload'
 
+import RecentContent from '@/components/RecentContent'
```

Comment:
For component imports, can we use named imports?

```suggestion
import { RecentContent } from '@/components/RecentContent'
```

for components, can we use named imports? (and default imports for page components).